client/my-sites/marketing/connections/service-connected-accounts.jsx

```diff
@@ -13,15 +13,16 @@ import { localize } from 'i18n-calypso';
 import { Button } from '@automattic/components';
 
 const SharingServiceConnectedAccounts = ( { children, connect, service, translate } ) => (
-	<div className="sharing-service-accounts-detail">
-		<ul className="sharing-service-connected-accounts">{ children }</ul>
-		{ 'publicize' === service.type && 'google_plus' !== service.ID && (
-			<Button onClick={ connect }>
-				{ translate( 'Connect a different account', {
-					comment: 'Sharing: Publicize connections',
-				} ) }
-			</Button>
-		) }
+	<div className="connections__sharing-service-accounts-detail">
+		<ul className="connections__sharing-service-connected-accounts">{ children }</ul>
+		{ ( 'publicize' === service.type || 'instagram-basic-display' === service.ID ) &&
+			'google_plus' !== service.ID && (
```

Comment:
Why do we have `google_plus` in here? Can we start ripping out this code?

Reply:
I guess we could but I wanted to avoid touching any existing behaviour as much as possible. 😅

Reply:
This is a great example of where we need to update Keyring and the related Publicize and Calypso code so that features, like the ability to have multiple accounts for a service, are not tied to the connection type. Ideally we'd said a config flag for the connection (e.g. `supports_multiple`) and everything would just work.